Add mnm_washu_2016

yichaoS opened the pull request:
Solving issue: https://github.com/cBioPortal/datahub/issues/337
Triage Instance:

Q1: 
The sample/patient count still isn't correct (116)

Q2:
Should we include Ampliseq and RMG panel?
The samples are not completely overlapped

jjgao commented on public/mixed_washu_nejm_2016/meta_study.txt:
```diff
@@ -0,0 +1,5 @@
+type_of_cancer: mixed
```
should it be `mnm` (Myeloid Neoplasm) since it is parent of both `aml` and `mds`?

jjgao commented on public/mixed_washu_nejm_2016/meta_study.txt:
```diff
@@ -0,0 +1,5 @@
+type_of_cancer: mixed
+cancer_study_identifier: mixed_washu_nejm_2016
```
maybe change this to `mnm_washu_2016` to follow our naming convention.

jjgao commented on public/mixed_washu_nejm_2016/meta_study.txt:
```diff
@@ -0,0 +1,5 @@
+cancer_study_identifier: mixed_washu_nejm_2016
+name: AML and MDS decitabine Response (WashU, 2016)  
+short_name: AML/MDS decitabine (WashU, 2016)
+description: Genomic profiling of 116 acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) patients receiving decitabine therapy. 
```
add pmid

yichaoS commented:
@jjgao renamed study and updated cancer type. I also added the issues to be resolved still in this study, just emailed the author, waiting for response.

yichaoS commented:
@jjgao There is a github issue created for this and the general checklist is included there: https://github.com/cBioPortal/datahub/issues/337. The issues I was referring to are some unclarity of the data itself and I just listed them in top comment section of this PR.

jjgao commented on public/mnm_washu_2016/meta_study.txt:
```diff
@@ -0,0 +1,6 @@
+name: AML and MDS decitabine Response (WashU, 2016)  
+short_name: AML/MDS decitabine (WashU, 2016)
+description: Genomic profiling of 116 acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) patients receiving decitabine therapy. 
+PMID: 27959731
```
I saw `pmid` is usually lower case in other studies.

jjgao commented on public/mnm_washu_2016/meta_study.txt:
```diff
@@ -3,4 +3,4 @@ cancer_study_identifier: mnm_washu_2016
 description: Genomic profiling of 116 acute myeloid leukemia (AML) and myelodysplastic syndromes (MDS) patients receiving decitabine therapy. 
-PMID: 27959731
\ No newline at end of file
+pmid: 27959731
```
I think we need a citation field too.
